storage/tianmu/handler/tianmu_handler.cpp

```diff
@@ -170,7 +170,23 @@ namespace {
 std::vector<bool> GetAttrsUseIndicator(TABLE *table) {
   int col_id = 0;
   std::vector<bool> attr_uses;
+  enum_sql_command sqlCommand = SQLCOM_END;
+  if (table->in_use && table->in_use->lex)
+    sqlCommand = table->in_use->lex->sql_command;
+  bool tianmuDeleteOrUpdate = sqlCommand == SQLCOM_DELETE || sqlCommand == SQLCOM_DELETE_MULTI ||
+                              sqlCommand == SQLCOM_UPDATE || sqlCommand == SQLCOM_UPDATE_MULTI;
```

Comment:
1. Uniform variables name, the same problem to `tianmuDeleteOrUpdate`.

2.Also, add `if` or `check` before the bool variable? Like `check `if_tianmu_delete_or_update`

3. Wrap the expression with brackets, e.g.: 
```
bool_value = (a==b) || (a==c) || ....
```